packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -122,17 +122,19 @@ interface EntryValidationTextProps {
 
 const EntryValidationText = ({ action, schema, entry }: EntryValidationTextProps) => {
   const { formatMessage } = useIntl();
-  const { validate } = unstable_useDocument(
+
+  const { validate, isLoading } = unstable_useDocument(
     {
       collectionType: schema?.kind ?? '',
       model: schema?.uid ?? '',
     },
     {
-      skip: !schema,
+      skip: true,
```

Comment:
🤔 what does skip do? Should it always be true?

Reply:
This is because how `useDocument` hook works. You can use this hook with documentId to get more details about an specific entry, but for this case we only want to get the validate function so you pass `skip: true` so it doesn't try to make any request to get more details about the document